spec/models/response_spec.rb

```diff
@@ -1,86 +1,140 @@
 describe Response do
-  let(:participant) { build(:participant, id: 1, user: build(:student, name: 'no name', fullname: 'no one')) }
   let(:participant2) { build(:participant, id: 2) }
   let(:assignment) { build(:assignment, id: 1, name: 'Test Assgt') }
-  let(:team) { build(:assignment_team) }
+  let(:participant) { build(:participant, id: 1, parent_id: 1, assignment: assignment, user: build(:student, name: 'no name', fullname: 'no one')) }
+  let(:team) { build(:assignment_team, id: 1) }
   let(:review_response_map) { build(:review_response_map, assignment: assignment, reviewer: participant, reviewee: team) }
-  let(:response) { build(:response, id: 1, map_id: 1, response_map: review_response_map, scores: [answer]) }
+  let(:answer5) { build(:answer5) }
+  let(:response) { build(:response, id: 1, map_id: 1, response_map: review_response_map, scores: [answer5]) }
+  let(:response1) { build(:response, id: 2, map_id: 1, response_map: review_response_map, scores: [answer2]) }
   let(:answer) { Answer.new(answer: 1, comments: 'Answer text', question_id: 1) }
   let(:answer2) { Answer.new(answer: 2, comments: 'Answer text', question_id: 2) }
   let(:question) { Criterion.new(id: 1, weight: 2, break_before: true) }
   let(:question2) { TextArea.new(id: 1, weight: 2, break_before: true) }
+  let(:question3) { build(:question3) }
```

Comment:
You could use one factory to create many objects with its own attributes. So you do not need to create `question3` factory, you could use `question` factory instead.